src/Utilities/ToolLocationHelper.cs

```diff
@@ -160,15 +160,15 @@ public enum VisualStudioVersion
         Version160,
 
         /// <summary>
-        /// Visual Studio "Dev17"
+        /// Visual Studio (Dev17)
```

Comment:
The planned name [is public](https://devblogs.microsoft.com/visualstudio/visual-studio-2022/).

```suggestion
        /// Visual Studio 2022 (Dev17)
```

Reply:
BenVillalobos made that exact comment in KirillOsenkov's PR:
https://github.com/dotnet/msbuild/pull/6336#discussion_r617709753

Somehow it wasn't merged, apparently 😄 

Thanks!